src/pkcs15init/pkcs15-oberthur-awp.c

```diff
@@ -996,6 +996,20 @@ awp_encode_cert_info(struct sc_pkcs15_card *p15card, struct sc_pkcs15_object *ob
 	/*
 	 * serial number
 	 */
+#if 1
+	
+	/* TODO the der encoding of a ANS1_INTEGER is a TLV, the original code only as using the 
+	 * i2c_ASN1_INTEGER which is not in OpenSSL 1.1  
```

Comment:
Although I appreciate this simplification, it's not very helpful when it is untested. Wouldn't it be safe to simply replace `i2c_ASN1_INTEGER` with `i2d_ASN1_INTEGER`?